packages/components/organisms/f-content-cards/src/components/ContentCards.js

```diff
@@ -35,10 +37,33 @@ export default {
     data: () => ({
         state: STATE_LOADING,
         cards: [],
-        errors: []
+        errors: [],
+        cardObserver: {}
     }),
 
+    watch: {
+        errors (errors) {
+            if (errors.filter(e => e === STATE_ERROR).length === this.adapters.length) {
+                this.state = STATE_ERROR;
+            }
+            if (errors.filter(e => e === STATE_NO_CARDS).length === this.adapters.length) {
+                this.state = STATE_NO_CARDS;
+            }
+            if (errors.length === this.adapters.length) {
+                this.state = STATE_ERROR;
+            }
+        }
+    },
+
     computed: {
+        xOffsets () {
+            return uniq(this.$children.map(({ $el }) => $el.offsetLeft)).sort((a, b) => a - b);
```

Comment:
A few things here:
1. Could you use a `Set` to make the array unique which would remove the need for the import?
2. I can't see lodash in the package.json, how are we importing it?
3. I think `.sort((a, b) => a - b)` behaves the same as just `.sort()`.

Reply:
1) Awesome idea, will make the change, 2) agreed, 3) good point